src/Microsoft.Framework.Logging.Console/ConsoleLoggerFactoryExtensions.cs

```diff
@@ -35,5 +37,23 @@ public static ILoggerFactory AddConsole(this ILoggerFactory factory, LogLevel mi
             factory.AddProvider(new ConsoleLoggerProvider((category, logLevel) => logLevel >= minLevel));
             return factory;
         }
+
+        /// <summary>
+        ///     Adds a console logger that is enabled as defined by the IConfiguration object
+        /// </summary>
+        /// <param name="configuration">IConfiguration object with filter attributes</param>
+        /// <returns></returns>
+        public static ILoggerFactory AddConsole(this ILoggerFactory factory, IConfiguration configuration)
+        {
+            factory.AddProvider(new ConsoleLoggerProvider((category, logLevel) =>
+            {
+                LogLevel minLevel;
+
+                return Enum.TryParse(configuration?.GetSubKey("ConsoleLogger")?.Get(category), out minLevel) &&
```

Comment:
It's perhaps conceivable that there could be, but I don't think anyone has really needed one yet. Even the case here doesn't really need it that badly. I'm not even sure it would benefit very much from it, would it?